cassandra/src/main/java/com/yahoo/ycsb/db/CassandraCQLClient.java

```diff
@@ -184,13 +209,13 @@ public void init() throws DBException {
         }
 
         Metadata metadata = cluster.getMetadata();
-        System.err.printf("Connected to cluster: %s\n",
+        logger.error("Connected to cluster: {}\n",
             metadata.getClusterName());
 
         for (Host discoveredHost : metadata.getAllHosts()) {
-          System.out.printf("Datacenter: %s; Host: %s; Rack: %s\n",
+          logger.info("Datacenter: {}; Host: {}; Rack: {}\n", new Object[] {
```

Comment:
you shouldn't need the array creation here.

Reply:
Unfortunately, slf4j's `info()` API has overloads for 1 and 2 arguments but not more. So the `Object` array is needed. https://www.slf4j.org/api/org/slf4j/Logger.html#info(java.lang.String,%20java.lang.Object...)

Reply:
The `Object ... arguments` version is varargs; you can pass it as many parameters as you want.

Reply:
There's no info(String format, Object...args) in the version of slf4j used in the project. That's why I had to use new Object[] {...}.

Reply:
oh! well that's easy enough to fix. #1149 updates us to current stable, rather than some version from 2012.

Reply:
Great. Will rebase and make the change after #1149 is merged.

Reply:
Rebased and made the suggested change.